src/intTest/resources/edifact/multi_transaction.1.dat

```diff
@@ -0,0 +1,55 @@
+UNB+UNOA:2+TES5+XX11+920116:1529+00000007'
+UNH+00000008+FHSREG:0:1:FH:FHS001'
+BGM+++507'
+NAD+FHS+XX1:954'
+DTM+137:199201171259:203'
+RFF+950:F4'
```

Comment:
These don't look like F4 Approvals. Are the actually amendments?

Reply:
Right. I've used that because appendix-j had only G2 examples which were currently unsupported by our code. I've added stub translator and now it works on G2